syncwatcher.go

```diff
@@ -434,6 +439,14 @@ func watchFolder(folder FolderConfiguration, stInput chan STEvent) {
 	}
 }
 
+func realPath(path string) (string, error) {
+	path, err := filepath.Abs(path)
+	if err != nil {
```

Comment:
@Unrud 
What is the reason for calling `filepath.Abs`?
I think the folder path provided by syncthing is absolut already (relative to what should it be?) and `filepath.Abs` prepends the "current working directory" to a relative path. The current working directory however can be anything, so this would result in undefined behavior. Am I missing something?